rfcs/formats/network/iin_method.md

```diff
@@ -0,0 +1,264 @@
+        ],
+        "updatePolicy": {
+          "id": "did:iin:<iin_name>:<network_name>#updatepolicy",
+          "controller": "did:iin:<iin_name>:<network_name>",
+          "type": "VerifiableCondition2021",
+          "conditionAnd": [{
+              "id": "did:iin:<iin_name>:<network_name>#updatepolicy-1",
+              "controller": "did:iin:<iin_name>:<network_name>",
+              "type": "VerifiableCondition2021",
+              "conditionOr": ["did:iin:<iin_name>:<network_participant_3>#key1",
+                "did:iin:<iin_name>:<network_participant_2>#key3"
+              ]
+            },
+            "did:iin:<iin_name>:<network_participant_1>#key1"
+          ]
+        }
+      },
+
+      {
+        "id": "did:iin:<iin_name>:<network_name>#fabriccerts",
+        "type": "DataplaneCredentials",
+        "controller": "did:iin:<iin_name>:<network_name>",
+        "FabricCredentials": {
+          "did:iin:<iin_name>:<network_participant_1>": "Certificate3_Hash",
+          "did:iin:<iin_name>:<network_participant_2>": "Certificate2_Hash",
+          "did:iin:<iin_name>:<network_participant_3>": "Certificate3_Hash"
+        }
+      }
+    ],
+    "authentication": [
+      "did:iin:<iin_name>:<network_name>#multisig"
+    ],
+    "relayEndpoints": [{
+        "hostname": "10.0.0.8",
+        "port": "8888"
+      },
+      {
+        "hostname": "10.0.0.9",
+        "port": "8888"
+      }
+
+    ]
+  },
+  "signatures": {
+    "did:iin:<iin_name>:<network_participant_1>": "...",
+    "did:iin:<iin_name>:<network_participant_2>": "...",
+    "did:iin:<iin_name>:<network_participant_3>": "..."
+  }
+}
+
+```
+
+How this request is created is out of the scopt of this specification. However, generally it is recommended that the signatures of the request are collected through a smart contract in the network blockchain itself.
+
+#####  Validation of Network DID creation request by the IIN
+
+The DID create method of the IIN registry must authenticate the a Network DID creation request differently from other DIDs. For a Network DID, the IIN first validates if the DID Document has the requisite properties: `networkParticipants`, and a verificationMethod of type `GroupMultiSig`. Additionally, the request must contain the signatures from all participants of the network authenticating it.
+
+The authentication process is carried out by validating signature from each participant in the `networkParticipants` as follows:
+
+1. The participant DID is obtained from `networkParticipants`.
+   - Example: `did:iin:<iin_name>:<network_participant_1>`
+2. From the `GroupMultiSig` verification, which verification method for the participant will be used is determined.
+   - Example: `did:iin:<iin_name>:<network_participant_1>#key1` indicates that key1 will be used which is specified by the fragment at the end of the URI.
+3. The DID Document of the participant is obtained by resolving the DID, and the required verification method is obtained.
```

Comment:
"obtained by resolving the DID" sounds vague and incomplete. Can you elaborate a bit more on the process?